app/src/app/list/polls-table/polls-table.component.sass

```diff
@@ -62,24 +66,23 @@ table
       margin-bottom: $medium-space
       padding: $small-space
       td
-        border: solid 1px #ddd
         padding: $tiny-space
         display: block
         text-align: right
         text-overflow: ellipsis
         overflow: hidden
         white-space: nowrap
+      td.title
+        border-bottom: solid 2px #ddd
+        text-align: left
+      td.round,  td.deadline,  td.terminated
+        font-size: smaller
+        text-align: right
         &:before
           content: attr(data-label)
           float: left
           font-weight: bold
```

Comment:
Sorry to not have noticed that before, but `bold` is an almost deprecated font-weight. You should use numeric values from 100 to 900. The font used here (Asap) is imported from Google Fonts with weights 400, 500 and 600 only. If you want a bolder weight, you must add it at line 10 of app/src/index.html.